src/js/sonifier.js

```diff
@@ -0,0 +1,402 @@
+        that.applier.change("sonifiedData",sonificationData);
+        that.events.onDataEntriesConvertedToSonificationData.fire();
+    };
+
+    // Creates a sonified data set based on unit divisors
+    // Longer tones represent the unit divisor, while a short tones is played
+    // for each remaining "1" of the remainder
+    floe.chartAuthoring.sonifier.unitDivisorSonificationStrategy = function(that, unitDivisor) {
+        var dataSet = that.model.dataSet;
+
+        var totalValue = that.model.total.value;
+        var sonificationData = [];
+        var synthOptions = that.options.synthOptions;
+        var playbackOptions = that.options.playbackOptions;
+
+        var noteDurationConfig = floe.chartAuthoring.sonifier.applyZoomToDurationConfig(synthOptions.noteDurationConfig,playbackOptions.zoom),
+            noteValueConfig = synthOptions.noteValueConfig,
+            envelopeDurationConfig = floe.chartAuthoring.sonifier.applyZoomToDurationConfig(synthOptions.envelopeDurationConfig,playbackOptions.zoom),
+            envelopeValuesConfig = synthOptions.envelopeValuesConfig;
+
+        fluid.each(dataSet, function(item,key) {
+            if(item.value !== null) {
+                var percentage = Number(floe.chartAuthoring.percentage.calculate(item.value, totalValue).toFixed(0));
+                var units = floe.chartAuthoring.sonifier.getSonificationUnits(percentage, unitDivisor);
+                var noteDurations = floe.chartAuthoring.sonifier.getSonificationNoteDurations(units, unitDivisor, noteDurationConfig);
+                var noteValues = floe.chartAuthoring.sonifier.getSonificationNoteValues(units, unitDivisor, noteValueConfig);
+                var envelopeDurations = floe.chartAuthoring.sonifier.getSonificationEnvelopeDurations(units, unitDivisor, envelopeDurationConfig);
+                var envelopeValues = floe.chartAuthoring.sonifier.getSonificationEnvelopeValues(envelopeDurations, envelopeDurationConfig, envelopeValuesConfig);
+                var d = {
+                    id: key,
+                    label: item.label,
+                    value: item.value,
+                    percentage: percentage,
+                    units: units,
+                    notes: {
+                        durations: noteDurations,
+                        values: noteValues
+                    },
+                    envelope: {
+                        durations: envelopeDurations,
+                        values: envelopeValues
+                    }
+                };
+                sonificationData.push(d);
+            }
+
+        });
+
+        return sonificationData;
+    };
+
+
+    floe.chartAuthoring.sonifier.getSonificationUnits = function(value, unitDivisor) {
+        var numberDivisors = Math.floor(value / unitDivisor);
+        var numberRemainders = value % unitDivisor;
+        var divisorArray =[];
+        var remainderArray = [];
+        for(var i=0; i <numberDivisors; i++) {
```

Comment:
@amb26 thanks to / in spite of the power of regular expressions, I've now corrected 419 whitespace errors after turning on `white: true` in .jshintrc